Command/DebugCommand.php

```diff
@@ -52,24 +53,23 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $stdout = $output;
```

Comment:
IMO, keeping ``$output`` ofr the raw output and changing the name for the styled output would be more readable if you deal with both.